src/data/data.js

```diff
@@ -78,6 +78,47 @@ const pageData = {
           },
         ],
       },
+      {
```

Comment:
Is this component visible on all pages?

Reply:
Not yet, because we have to wait for the update in the data.js, while this happens the footer component is within the events component